src/main.js

```diff
@@ -34,7 +34,11 @@ const BookmarksApp = (global.Bookmarks = new Vue({
 }))
 
 if ('serviceWorker' in navigator) {
-	navigator.serviceWorker.register(generateUrl('/apps/bookmarks/service-worker.js'))
+	const webRootWithIndexPHP = getRootUrl() + '/index.php'
+	const doesURLContainIndexPHP = window.location.pathname.startsWith(webRootWithIndexPHP)
+	navigator.serviceWorker.register(generateUrl('/apps/bookmarks/service-worker.js', {}, {
+		noRewrite: doesURLContainIndexPHP,
```

Comment:
```suggestion
		noRewrite: doesURLContainIndexPHP,
```
we may need to invert the logic here to make it work. not sure though...

Reply:
or we simply always set it to true, WDYT?

Reply:
Will do this...

Reply:
done